.gitignore

```diff
@@ -13,7 +13,6 @@
 /dist
 /mkosi.build
 /mkosi.egg-info
-/mkosi.extra
```

Comment:
I think it might be nicer for contributors if we kept this and instead allowed the other files instead, i.e.
```
!mkosi.extra/usr/lib/systemd/mkosi-check-and-shutdown.sh
!mkosi.extra/usr/lib/systemd/mkosi-check-and-shutdown.sh
```